bin/index.js

```diff
@@ -75,15 +78,15 @@ const startCommand = [
   `-c 'bgYellow.bold,bgMagenta.bold,bgCyan.bold,bgGreen.bold'`,
 
   // start the reverse proxy
-  `"node ./src/proxy.js"`,
+  `"node ./src/proxy.js --artifacts ${app_artifact_location}"`,
```

Comment:
app_artifact_location is available via `SWA_EMU_APP_LOCATION` ([here](https://github.com/manekinekko/swa-emulator/blob/main/bin/index.js#L65))